packages/node-app/src/tests/auth.controller.test.ts

```diff
@@ -0,0 +1,204 @@
+import fs from 'fs';
+import mongoose from 'mongoose';
+import request from 'supertest';
+
+import { app } from '../app';
+import { giveawaysContract } from '../contracts';
+import { Giveaway } from '../models/giveaway.model';
+import { Location } from '../models/location.model';
+import { Unit, User, UserRole } from '../models/user.model';
+import { getUserInfo } from '../utils/auth.util';
+
+const userInfo = {
+  email: 'example@domain.com',
+  name: 'name',
+  picture: 'picture',
+  unit: Unit.NODE,
+};
+
+giveawaysContract.methods.createGiveaway = jest
+  .fn()
+  .mockReturnValue({ send: () => ({}) });
+
+giveawaysContract.methods.addParticipant = jest
+  .fn()
+  .mockReturnValue({ send: () => ({}) });
+
+jest.mock('../utils/auth.util.ts', () => ({
+  getUserInfo: jest.fn(),
+}));
+
+beforeAll(async () => {
+  await mongoose.connect(process.env.TEST_DATABASE_URI);
+});
+
+afterAll(async () => {
+  await mongoose.connection.dropDatabase();
+  await mongoose.connection.close();
+});
+
+afterEach(async () => {
+  await User.deleteMany({});
+  await Location.deleteMany({});
+  await Giveaway.deleteMany({});
+  jest.clearAllMocks();
+});
+
+describe('POST /login', () => {
+  it('login new user', async () => {
```

Comment:
Jest follows a `it should ...` naming convention for tests. So test description should be readable, like `it should allow a new user to login`
```suggestion
  it('should allow new user to login', async () => {
```